sql/7_0_3-to-7_0_4_upgrade.sql

```diff
@@ -612,7 +612,296 @@ VALUES ('sdoh_instruments', 'hunger_vital_sign', 'Hunger Vital Sign (2-item)', 1
+    ('DEM','related_workphone_2',@newgrp,'Work Phone',230,2,1,20,63,'',1,1,'','','Work Phone',0,'','F','','',''),
+    ('DEM','related_email_2',@newgrp,'Email',240,2,1,20,63,'',1,1,'','','Related Email Address',0,'','F','','',''),
+    ('DEM','related_static_2',@newgrp,'Related Person',245,31,1,0,0,'',1,3,'','["K"]','Patient Third Related Person',0,'','F','','',''),
+
+    ('DEM','related_firstname_3',@newgrp,'First Name',250,2,1,25,63,'',1,1,'','["K"]','Related First Name',0,'','F','','',''),
+    ('DEM','related_lastname_3',@newgrp,'Last Name',260,2,1,25,63,'',1,1,'','','Last Name',0,'','F','','',''),
+    ('DEM','related_relationship_3',@newgrp,'Relationship',270,1,1,0,0,'personal_relationship',1,1,'','','Relationship',0,'','F','','',''),
+    ('DEM','related_sex_3',@newgrp,'Sex',280,1,1,0,0,'sex',1,1,'','','Sex',0,'','F','','',''),
+    ('DEM','related_address_3',@newgrp,'Address',290,2,1,25,63,'',1,1,'','','Address',0,'','F','','',''),
+    ('DEM','related_city_3',@newgrp,'City',300,2,1,15,63,'',1,1,'','','City',0,'','F','','',''),
+    ('DEM','related_state_3',@newgrp,'State',310,26,1,0,0,'state',1,1,'','','State',0,'','F','','',''),
+    ('DEM','related_postalcode_3',@newgrp,'Postal Code',320,2,1,6,63,'',1,1,'','','Postal Code',0,'','F','','',''),
+    ('DEM','related_country_3',@newgrp,'Country',330,26,1,0,0,'country',1,1,'','','Country',0,'','F','','',''),
+    ('DEM','related_phone_3',@newgrp,'Phone',340,2,1,20,63,'',1,1,'','','Phone',0,'','F','','',''),
+    ('DEM','related_workphone_3',@newgrp,'Work Phone',350,2,1,20,63,'',1,1,'','','Work Phone',0,'','F','','',''),
+    ('DEM','related_email_3',@newgrp,'Email',360,2,1,20,63,'',1,1,'','','Related Email Address',0,'','F','','','');
+#EndIf
+
+-- =========================
+-- DEM Layout: Tribal Affiliations
+-- Resequence group by order of 10
+-- =========================
+#IfNotRow2D layout_options form_id DEM field_id tribal_affiliations
+SET @group_id =(SELECT `group_id` FROM layout_options WHERE field_id='religion' AND form_id='DEM');
+SET @seq_start := 0;
+UPDATE `layout_options` SET `seq` = (@seq_start := @seq_start+1)*10 WHERE group_id = @group_id AND form_id='DEM' ORDER BY `seq`;
+SET @seq_add_to = (SELECT seq FROM layout_options WHERE group_id = @group_id AND field_id='religion' AND form_id='DEM');
+INSERT INTO `layout_options` (`form_id`, `field_id`, `group_id`, `title`, `seq`, `data_type`, `uor`, `fld_length`, `max_length`, `list_id`, `titlecols`, `datacols`, `default_value`, `edit_options`, `description`, `fld_rows`, `list_backup_id`, `source`, `conditions`, `validation`, `codes`) VALUES ('DEM','tribal_affiliations',@group_id,'Tribal Affiliations',@seq_add_to+10,1,1,0,0,'tribal_affiliations',1,1,'','','Tribal Affiliations entries',0,'','F','','','');
+ALTER TABLE `patient_data` ADD `tribal_affiliations` TEXT;
+#Endif
+
+-- =========================
+-- form_history_sdoh: grouped adds
+-- =========================
+#IfTable form_history_sdoh
```

Comment:
Looks like this table in database.sql, so would instead check if the `instrument_score` column is missing and then run block of code adding the columns in standard fashion.